navbarJS.js

```diff
@@ -1,161 +0,0 @@
-var toggle =document.querySelector('.toggle');
-var menue =document.querySelector('.menue');
-
-function toggleMenue() {
-  if (menue.classList.contains("active")) {
-    menue.classList.remove("active");
-     
-    // adds the menu (hamburger) icon  
-   toggle.innerHTML = `<img id="icon" src="menue.png" class="icon" >`;
-} else {
-    menue.classList.add("active");
-     
-    // adds the close (x) icon
-   toggle.innerHTML =  `<img id="icon" src="close.png" class="icon" >`;
-}
-}
-
-
-
-
-//Filter part
-
-
-var val = document.getElementById("list1").value;
-var list2 = document.getElementById("list2");
-let arrayList= [];
-
-/*let arrayListOb = [{name:"HTML" , isDone:false},
-                     {name:"JavaScript" , isDone:false},
-                     {name:"CSS" , isDone:false}];
-                             console.log(arrayListOb[2].name);        */
- let arrayListOb=[];    
-    let HTMLOb = {name:"HTML" , isDone:false};
-    let CSSOb ={name:"JavaScript" , isDone:false};
-    let JavaScriptOb= {name:"CSS" , isDone:false};
- 
-
-//arrayListOb.unshift(JavaScriptOb); localStorage.setItem("arrayListOb", JSON.stringify(arrayListOb));
-//arrayListOb.unshift(CSSOb); localStorage.setItem("arrayListOb", JSON.stringify(arrayListOb));
-//arrayListOb.unshift(HTMLOb); localStorage.setItem("arrayListOb", JSON.stringify(arrayListOb));
-arrayListOb = JSON.parse(localStorage.getItem('arrayListOb'));
-
-
-arrayList = JSON.parse(localStorage.getItem('arrayList1'));
-
-doneStyle= JSON.parse(localStorage.getItem('doneStyle1'));
-//window.localStorage.clear();
-//arrayList.unshift("JavaScript"); localStorage.setItem("arrayList1", JSON.stringify(arrayList));
-//arrayList.unshift("CSS"); localStorage.setItem("arrayList1", JSON.stringify(arrayList));
-//arrayList.unshift("HTML"); localStorage.setItem("arrayList1", JSON.stringify(arrayList));
-
-let filterDiv= document.getElementById("allItems");
-
-
-
-for(let i=0;i< arrayListOb.length; i++){
```

Comment:
you can use arrayListOb.forEach , read about it

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach